<ui version="4.0" >
 <class>ImportWiktionaryDlg</class>
 <widget class="QWidget" name="ImportWiktionaryDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLabel" name="lbDesc" >
     <property name="text" >
      <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Das Wiktionary ist ein freies Wörterbuch der Wikimedia foundation und veröffentlicht Dumps seiner Datenbank die frei heruntergeladen werden können.&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Derzeit werden nur deutsche Wiktionaries unterstützt.&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
     <property name="openExternalLinks" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbImportLocal" >
     <property name="text" >
      <string>Lokale Datei:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Datei (.xml / .xml.bz2):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="SimonFileSelector" name="leWikiPath" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="rbImportRemote" >
     <property name="text" >
      <string>Direkt herunterladen:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="SimonListWidget" name="lwRemoteList" />
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SimonFileSelector</class>
   <extends>QLineEdit</extends>
   <header>simonfileselector.h</header>
  </customwidget>
  <customwidget>
   <class>SimonListWidget</class>
   <extends>QListWidget</extends>
   <header>simonlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
